tests/wpt/web-platform-tests/url/url-domainToUnicode.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<meta charset="utf-8">
+<title></title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+function runURLDomainToUnicodeTests() {
+  test(function() {
+    var domain = 'example.org'
+    assert_true(URL.domainToUnicode(domain) === domain, 'Ascii domain should be parsed correctly.')
+  }, 'URL.domainToUnicode valid input')
+
+  test(function() {
+    var domain = 'xn--maana-pta.com'
+    assert_true(URL.domainToUnicode(domain) === 'mañana.com', ' Ascii encoded domain should be parsed correctly.')
+  }, 'URL.domainToUnicode valid encoded input')
+
+  test(function() {
+    var domain = 'http://not.a.domain'
+    assert_true(URL.domainToUnicode(domain) === "", 'Invalid domain should be return an empty string.')
+  }, 'URL.domainToUnicode invalid input')
+
+}
+runURLDomainToUnicodeTests()
```

Comment:
I don't think we need to define this function at all. Can we just remove it and have the `test` functions at the top level?